src/components/AddNewBook.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import Dropdown from 'react-dropdown';
+import 'react-dropdown/style.css';
+
```

Comment:
- [ ] I noticed that your app does not add books via the API because you did not refactor the app as required 😢  . According to the project requirement, adding the book function should be refactored,  return a promise and add a new book via API. Please fix it in order to make your app works 👍 

- [ ] I noticed that we are not able to remove books via the API because you did not refactor the app as required 😢  . According to the project requirement, remove a book function should be refactored,  return a promise and add a new book via API. Please fix it in order to make your app works 👍